app/src/main/AndroidManifest.xml

```diff
@@ -13,12 +13,10 @@
 
     <application
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: GoogleAppIndexingWarning
**Severity**: WARN
**File**: C:/Users/anisart/.jenkins/workspace/test3/app/src/main/AndroidManifest.xml L14

Missing support for Firebase App Indexing&#10;App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.&#10;Adds URLs to get your app into the Google index, to get installs and traffic to your app from Google Search.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1066774692>*